README.md

```diff
@@ -1,85 +1,78 @@
+| cosmos           | [azure-spring-boot-starter-cosmos:3.13.0]                                                        | [cosmos-multi-database-single-account](cosmos/azure-spring-boot-starter-cosmos/cosmos-multi-database-single-account)              |
+| cosmos           | [azure-spring-boot-starter-cosmos:3.13.0]                                                        | [cosmos](cosmos/azure-spring-boot-starter-cosmos/cosmos)                                                                          |
+| eventhubs        | [azure-spring-cloud-starter-eventhubs-kafka:2.13.0]                                              | [eventhubs-kafka](eventhubs/azure-spring-cloud-starter-eventhubs-kafka/eventhubs-kafka)                                           |
+| eventhubs        | [azure-spring-cloud-starter-eventhubs:2.13.0]                                                    | [eventhubs-integration](eventhubs/azure-spring-cloud-starter-eventhubs/eventhubs-integration)                                     |
+| eventhubs        | [azure-spring-cloud-starter-eventhubs:2.13.0]                                                    | [eventhubs-messaging](eventhubs/azure-spring-cloud-starter-eventhubs/eventhubs-messaging)                                         |
+| eventhubs        | [azure-spring-cloud-starter-eventhubs:2.13.0]                                                    | [eventhubs-operation](eventhubs/azure-spring-cloud-starter-eventhubs/eventhubs-operation)                                         |
+| eventhubs        | [azure-spring-cloud-stream-binder-eventhubs:2.13.0]                                              | [eventhubs-binder](eventhubs/azure-spring-cloud-stream-binder-eventhubs/eventhubs-binder)                                         |
+| eventhubs        | [azure-spring-cloud-stream-binder-eventhubs:2.13.0]                                              | [eventhubs-multibinders](eventhubs/azure-spring-cloud-stream-binder-eventhubs/eventhubs-multibinders)                             |
+| keyvault         | [azure-spring-boot-starter-keyvault-certificates:3.13.0]                                          | [keyvault-certificates-client-side](keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side)   |
+| keyvault         | [azure-spring-boot-starter-keyvault-certificates:3.13.0]                                          | [keyvault-certificates-server-side](keyvault/azure-security-keyvault-jca/run-with-command-line-server-side)   |
+| keyvault         |                                                                                                  | [run-with-command-line-server-side](keyvault/azure-security-keyvault-jca/run-with-command-line-client-side)   |
+| keyvault         |                                                                                                  | [run-with-command-line-client-side](keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side)   |
```

Comment:
I think you delete `azure-security-keyvault-jca` because it's not a `Azure Spring Starter Dependency`.
But `azure-messaging-servicebus` / `azure-media` are not  `Azure Spring Starter Dependency`, too, you kept them.

How about changing `Azure Spring Starter Dependency` to `Related Dependency`?